test/unit/rubygem_searchable_test.rb

```diff
@@ -15,7 +15,8 @@ class RubygemSearchableTest < ActiveSupport::TestCase
         number: "1.0.1",
         rubygem: @rubygem,
         summary: "some summary",
-        description: "some description")
+        description: "some description",
+        metadata: { "funding_uri" => "http://example.com" })
```

Comment:
you will need to specify all *_uri keys in this. 
sorry, this is a bit confusing, this test should have been updated when we migrated to metadata links.

Reply:
I actually can't run tests locally so hopefully this works.

Reply:
let us know if something is not working on dev set up, we have a docker compose file for all services deps.